src/main/java/net/minestom/server/item/ItemMetaBuilder.java

```diff
@@ -85,10 +87,11 @@
     public @NotNull ItemMetaBuilder lore(@NotNull List<@NotNull Component> lore) {
         this.lore = new ArrayList<>(lore);
         handleCompound("display", nbtCompound -> {
-            final NBTList<NBTString> loreNBT = new NBTList<>(NBTTypes.TAG_String);
-            for (Component line : lore) {
-                loreNBT.add(new NBTString(GsonComponentSerializer.gson().serialize(line)));
-            }
+            final NBTList<NBTString> loreNBT = NBT.List(NBTType.TAG_String,
+                    lore.stream()
+                            .map(line -> new NBTString(GsonComponentSerializer.gson().serialize(line)))
+                            .collect(Collectors.toList())
```

Comment:
All those could be replaced by the newer `#toList()` allocating an immutable list

Reply:
Will fix in next commit